client/src/components/Onboarding/OnboardingStatus.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Pill, LoadingSpinner } from '_common';
+import { stepPropType } from './OnboardingPropTypes';
+import './OnboardingStatus.module.scss';
+
+const getContents = step => {
+  let type = '';
+  switch (step.state) {
+    case 'processing':
+    case 'pending':
+      type = 'normal';
+      break;
+    case 'failed':
+      type = 'danger';
+      break;
+    case 'staffwait':
+    case 'userwait':
+      type = 'alert';
+      break;
+    case 'completed':
+      type = 'success';
+      break;
+    default:
+      type = 'normal';
+  }
+  if ('customStatus' in step) {
+    return <Pill type={type}>{step.customStatus}</Pill>;
+  }
+  switch (step.state) {
+    case 'pending':
+      return <Pill type={type}>Preparing</Pill>;
+    case 'staffwait':
+      return <Pill type="normal">Waiting for Staff Approval</Pill>;
+    case 'userwait':
+      return null;
+    case 'failed':
+      return <Pill type={type}>Unsuccessful</Pill>;
+    case 'completed':
+      return <Pill type={type}>Completed</Pill>;
+    case 'processing':
+      return (
+        <span styleName="processing">
+          <Pill type={type}>Processing</Pill>
+          <LoadingSpinner placement="inline" />
+        </span>
+      );
+    default:
+      return <span>{step.state}</span>;
+  }
+};
+
+const OnboardingStatus = ({ step }) => {
+  return <span styleName="root">{getContents(step)}</span>;
```

Comment:
This should return nothing if `getContents` returns `null`, so as to avoid any styling applied to this component appearing in the dom (it currently applies a blank space)
<img width="199" alt="Screen Shot 2020-11-05 at 12 44 32 AM" src="https://user-images.githubusercontent.com/20326896/98206786-1eac4700-1f00-11eb-8ab0-e6135b2274fa.png">